tutorials/download_tinyllama.md

```diff
@@ -14,19 +14,19 @@ python scripts/download.py | grep TinyLlama
 which will print
 
 ```text
-TinyLlama/TinyLlama-1.1B-intermediate-step-955k-token-2T
-TinyLlama/TinyLlama-1.1B-Chat-v0.6
+TinyLlama/TinyLlama-1.1B-intermediate-step-1431k-3T
+TinyLlama/TinyLlama-1.1B-chat-v1.0
```

Comment:
Take a look at the changed lines and check that `Chat` is used instead of `chat`.